angular-demo/package.json

```diff
@@ -25,11 +25,11 @@
     "@angular/platform-browser-dynamic": "8.2.5",
     "@mapbox/mapbox-gl-draw": "^1.1.2",
     "@mapbox/mapbox-gl-geocoder": "^4.4.1",
-    "@pxblue/angular-components": "1.1.1",
-    "@pxblue/colors": "1.0.13",
+    "@pxblue/angular-components": "3.0.1",
+    "@pxblue/angular-themes": "5.1.2",
+    "@pxblue/colors": "3.0.0",
     "@pxblue/mapbox": "^1.0.1",
-    "@pxblue/ng-progress-icons": "1.1.4",
-    "@pxblue/themes": "2.1.0",
+    "@pxblue/ng-progress-icons": "1.2.1",
```

Comment:
Use version ranges with carets ^ instead of fixed versions.

Reply:
update coming